### PR TITLE
chore: add config examples

### PR DESCRIPTION
**Pull request by duguorong009**
## Changes
- add examples of `.env` & `config.toml`
- improve the README in `contracts`

## Reminders
- [x] Add a prefix to PR: `feat:` (for features), `chore:` (for chores/refactoring), `fix:` (for bug fixes), or any from: https://www.conventionalcommits.org/en/v1.0.0/#summary
- [x] Assign reviewers
- [ ] Link related issues
- [x] Make it draft if it is still WIP
- [x] Write unit tests for your code if possible
- [x] Do manual testing and make sure your changes don't break the rest of the codebase

### CODE REVIEW

File: client/.env.example [hunk 1]
```diff
@@ -1 +1 @@
-SC_CLIENT_WALLET_SECRET_KEY=aabbccdd... # smart contract client wallet private key
+SC_CLIENT_WALLET_SECRET_KEY=ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 # anvil 1st private key; Need to be changed with your own key if not in anvil
```
Comment (astralblue) [inline client/.env.example:1]: Put description before the parameter:
```suggestion
# anvil 1st private key; Need to be changed with your own key if not in anvil
SC_CLIENT_WALLET_SECRET_KEY=ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
```

File: client/config.toml.dev [hunk 2]
```diff
@@ -0,0 +1,8 @@
+contract_address = "0xaabbcc..."
+chain_rpc_url = "https://rpc-amoy.polygon.technology/" # polygon amoy testnet
+chain_id = 80002
+
+openrank_rpc_url="https://or-dev-prod.k3l.io/"
```
Comment (astralblue) [inline client/config.toml.dev:5]:
```suggestion
# OpenRank block builder JSON-RPC endpoint
openrank_rpc_url="https://or-dev-prod.k3l.io/API_KEY"  # contact ek@karma3labs.com for API key
```

File: contracts/script/DeployComputeManager.s.sol [hunk 3]
```diff
@@ -15,7 +15,7 @@ contract ComputeManagerScript is Script {
 
         address[] memory submitters = new address[](3);
         submitters[0] = address(bytes20(bytes(hex"b79aafc95c8866e65ed51a7856e75587feb481ff")));
-        submitters[1] = address(bytes20(bytes(hex"4d40f98096410aa4235292343cc5fa559afcc780")));
+        submitters[1] = address(bytes20(bytes(hex"f39Fd6e51aad88F6F4ce6aB8827279cffFb92266")));
```
Comment (astralblue) [inline contracts/script/DeployComputeManager.s.sol:18]: Is it possible to segregate these hard-coded addresses—and other parameters that the deployer needs to modify at compile time—into a separate source file, with documentation comments?  Then we can tell people in README to take a look at that file.  C/C++ projects often use `config.h` for that purpose.

We can also give multiple example versions of that file and remove the real file, like we do for `config.toml.dev` ⇒ `config.toml`.

**Review by astralblue — APPROVED** [review]
Overall LGTM.  Below please find a few minor cosmetic comments (OK if not resolved) and a follow-up refactor suggestion for smart contract build parameter handling.

File: contracts/script/DeployComputeManager.s.sol (hunk 3 above)
Comment (duguorong009) [reply contracts/script/DeployComputeManager.s.sol]: Thx for valuable comments! @astralblue 
I've tried to improve the overall PR.
Pls review the PR, again.

**Review by lazovicff — APPROVED** [review]
Approved now, I will test later

**Review by astralblue — APPROVED** [review]
LGTM, thanks for addressing the refactoring need!  TIL `vm.envAddress[]` :smile: :heart:  Foundry

**Pull request merged**